### PR TITLE
ip-route-get: add page

### PR DESCRIPTION
**Pull request by ryuheechul**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/ip-route-get.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# ip route get
+
+> Get a single route - this command gets a single route to a destination and prints its contents exactly as the kernel sees it.
```
Comment (spageektti) [inline pages/linux/ip-route-get.md:3]:
```suggestion
> Get a single route to a destination and print its contents exactly as the kernel sees it.
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution.

File: pages/linux/ip-route-get.md (hunk 1 above)
Comment (cyqsimon) [reply pages/linux/ip-route-get.md:3]: A brief explanation for the author:

The original description seems to be posted verbatim from the manual. Here at tldr though, the description should be adapted to fit [the style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).

File: pages/linux/ip-route-get.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`ip route get 127.0.0.1 from 127.0.0.2`
+
+- Print route from 127.0.0.1 to 0.0.0.0:
+
+`ip route get 0.0.0.0 from 127.0.0.1`
+
+- Print route from 0.0.0.0 to 127.0.0.1:
+
+`ip route get 127.0.0.1 from 0.0.0.0`
```
Comment (cyqsimon) [inline pages/linux/ip-route-get.md:24]: These examples feel repetitive and redundant.

I think they should either be removed, or be replaced with examples that showcase some other potentially useful options (e.g. `iif`, `oif`, `sport`, `dport`)

**Review by cyqsimon — CHANGES_REQUESTED** [review]
Thanks for your contribution. A few opinions:

File: pages/linux/ip-route-get.md (hunk 2 above)
Comment (ryuheechul) [reply pages/linux/ip-route-get.md:24]: Thanks for the comment. It maybe repetitive in the perspective of the command itself but I chose the combination of addresses that help you get the idea of what routes would work or what kind of paths that user should trace. But I see that it maybe too opinionated.

I'm not familiar with the usages with `iif`, `oif`, `sport`, `dport`. Feel free to suggests a edits or feel free to overwrite with another PR :)

File: pages/linux/ip-route-get.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
```
Comment (spageektti) [inline pages/linux/ip-route-get.md:24]:
```suggestion
- Print route to a destination:

`ip route get {{1.1.1.1}}`

- Print route to a destination from source:

`ip route get {{destination}} from {{source}}`
```

**Review by spageektti — CHANGES_REQUESTED** [review]
Please check my suggestion

File: pages/linux/ip-route-get.md (hunk 3 above)
Comment (ryuheechul) [reply pages/linux/ip-route-get.md]: Thanks for the suggestion!

**Review by cyqsimon — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**